forms/common-themes/src/main/resources/theme/admin/base/resources/js/services.js

```diff
@@ -1068,9 +1070,9 @@ module.factory('PasswordPolicy', function() {
             var re = /(\w+)\(*(\d*)\)*/;
 
             var policyEntry = re.exec(policyToken);
-
-            policies.push({ name: policyEntry[1], value: parseInt(policyEntry[2]) });
-
+            if (null !== policyEntry) {
```

Comment:
This is a bug I found while testing this: adding a policy, saving, coming back to this screen caused the application to "break" with a JavaScript error. This is because the policy string is built as "hashIterations(1) and", so, JavaScript thinks there are 2 elements. On the Java side of the code, this is an expected situation and handled accordingly (by ignoring the last element). So, I'm doing the same here in JavaScript.